caffe2/operators/experimental/c10/cpu/stop_gradient_cpu.cc

```diff
@@ -18,11 +18,23 @@ void stop_gradient_op_cpu_impl(
     output.CopyFrom(input);
   }
 }
+
+static auto registry = c10::RegisterOperators().op(
+    FunctionSchema(
+        "_c10_experimental::StopGradient",
+        "",
+        (std::vector<c10::Argument>{c10::Argument("input"),
+                                    c10::Argument("output")}),
+        (std::vector<c10::Argument>{})),
```

Comment:
This is pretty wordy way to specify arguments...

Reply:
see comment above